docs/tutorial/05-loops-conditions-existing.md

```diff
@@ -83,6 +83,11 @@ resource blob 'Microsoft.Storage/storageAccounts/blobServices/containers@2019-06
   name: '${stg.name}/default/${name}'
   // dependsOn will be added when the template is compiled
 }]
+
+// with index
+resource blob2 'Microsoft.Storage/storageAccounts/blobServices/containers@2019-06-01' = [for (name, index) in containerNames: {
```

Comment:
I think it's a little confusing to have two blocks in the code that will create containers. This example, if I include both will create six containers total. I'd either recommend splitting this into a separate code block so it's more clear that this is an alternative, or add a link to the text you added above to teach people about the index syntax.